1756483774.8145065 [INFO] [launch]: All log files can be found below /home/rpp/rpp_logs/2025-08-30-00-09-34-814042-rpp-Default-string-11243
1756483774.8145866 [INFO] [launch]: Default logging verbosity is set to INFO
1756483774.8453743 [INFO] [iat_publish-1]: process started with pid [11244]
1756483774.8454609 [INFO] [tts_subscribe-2]: process started with pid [11246]
1756483774.8455069 [INFO] [voice_control-3]: process started with pid [11248]
1756483774.8455420 [INFO] [voice_dialog-4]: process started with pid [11250]
1756483774.8699086 [iat_publish-1] [INFO] [1756483774.869679066] [iat_node]: MSPLogin success.
1756483775.0907257 [voice_dialog-4] [INFO] [1756483775.090557980] [voice_dialog_node]: 语音对话节点已启动
1756483775.1280074 [voice_control-3] [WARN] [1756483775.127841323] [voice_control_node]: 尚未获取 base_angle，无法转向
1756483775.1335137 [voice_control-3] [INFO] [1756483775.133321791] [voice_control_node]: 已加载 JSON 文件: /home/rpp//rpp_data/config/voice/voice_pose.json
1756483775.1337321 [voice_control-3] [INFO] [1756483775.133496806] [voice_control_node]: 等待 TF [mic_link -> base_link] 可用...
1756483775.1346679 [voice_control-3] [INFO] [1756483775.134607527] [voice_control_node]: TF 尚未可用，继续等待...
1756483775.6359210 [voice_control-3] [INFO] [1756483775.635764468] [voice_control_node]: TF 尚未可用，继续等待...
1756483776.1386414 [voice_control-3] [INFO] [1756483776.138472316] [voice_control_node]: TF 尚未可用，继续等待...
1756483776.6398840 [voice_control-3] [INFO] [1756483776.639732716] [voice_control_node]: TF 尚未可用，继续等待...
1756483777.1411564 [voice_control-3] [INFO] [1756483777.140982654] [voice_control_node]: TF 尚未可用，继续等待...
1756483777.6427345 [voice_control-3] [INFO] [1756483777.642591533] [voice_control_node]: TF 获取成功 
1756483777.6431358 [voice_control-3] [INFO] [1756483777.643070237] [voice_control_node]: TTS 成功播报：成功加载 2 个点位
1756483820.2588527 [WARNING] [launch]: user interrupted with ctrl-c (SIGINT)
1756483820.2600896 [voice_dialog-4] Traceback (most recent call last):
1756483820.2601681 [voice_dialog-4]   File "/home/rpp/rpp_ws/install/xf_voice/lib/xf_voice/voice_dialog", line 138, in <module>
1756483820.2602022 [voice_dialog-4]     main()
1756483820.2602291 [voice_dialog-4]   File "/home/rpp/rpp_ws/install/xf_voice/lib/xf_voice/voice_dialog", line 133, in main
1756483820.2602539 [voice_dialog-4]     rclpy.spin(node)
1756483820.2602766 [voice_dialog-4]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 226, in spin
1756483820.2602975 [voice_dialog-4]     executor.spin_once()
1756483820.2603207 [voice_dialog-4]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 739, in spin_once
1756483820.2603416 [voice_dialog-4]     self._spin_once_impl(timeout_sec)
1756483820.2603676 [voice_dialog-4]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 728, in _spin_once_impl
1756483820.2603900 [voice_dialog-4]     handler, entity, node = self.wait_for_ready_callbacks(timeout_sec=timeout_sec)
1756483820.2604103 [voice_dialog-4]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 711, in wait_for_ready_callbacks
1756483820.2604311 [voice_dialog-4]     return next(self._cb_iter)
1756483820.2604554 [voice_dialog-4]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 608, in _wait_for_ready_callbacks
1756483820.2604895 [voice_dialog-4]     wait_set.wait(timeout_nsec)
1756483820.2605100 [voice_dialog-4] KeyboardInterrupt
1756483820.2606416 [iat_publish-1] [INFO] [1756483820.258915241] [rclcpp]: signal_handler(signum=2)
1756483820.2607517 [tts_subscribe-2] [INFO] [1756483820.258919113] [rclcpp]: signal_handler(signum=2)
1756483820.2608750 [voice_control-3] Traceback (most recent call last):
1756483820.2609124 [voice_control-3]   File "/home/rpp/rpp_ws/install/xf_voice/lib/xf_voice/voice_control", line 572, in <module>
1756483820.2609377 [voice_control-3]     main()
1756483820.2609699 [voice_control-3]   File "/home/rpp/rpp_ws/install/xf_voice/lib/xf_voice/voice_control", line 566, in main
1756483820.2610185 [voice_control-3]     rclpy.spin(node)
1756483820.2610428 [voice_control-3]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 226, in spin
1756483820.2610643 [voice_control-3]     executor.spin_once()
1756483820.2610855 [voice_control-3]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 739, in spin_once
1756483820.2611134 [voice_control-3]     self._spin_once_impl(timeout_sec)
1756483820.2611468 [voice_control-3]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 728, in _spin_once_impl
1756483820.2611692 [voice_control-3]     handler, entity, node = self.wait_for_ready_callbacks(timeout_sec=timeout_sec)
1756483820.2612014 [voice_control-3]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 711, in wait_for_ready_callbacks
1756483820.2612209 [voice_control-3]     return next(self._cb_iter)
1756483820.2612422 [voice_control-3]   File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 608, in _wait_for_ready_callbacks
1756483820.2612631 [voice_control-3]     wait_set.wait(timeout_nsec)
1756483820.2612939 [voice_control-3] KeyboardInterrupt
1756483820.2815783 [ERROR] [voice_dialog-4]: process has died [pid 11250, exit code -2, cmd '/home/rpp/rpp_ws/install/xf_voice/lib/xf_voice/voice_dialog --ros-args -r __node:=voice_dialog_node --params-file /tmp/launch_params_rebohbv5 --params-file /tmp/launch_params_qz_z3fpb --params-file /tmp/launch_params_kth7jde3'].
1756483820.2876732 [ERROR] [voice_control-3]: process has died [pid 11248, exit code -2, cmd '/home/rpp/rpp_ws/install/xf_voice/lib/xf_voice/voice_control --ros-args -r __node:=voice_control_node'].
1756483820.3659563 [INFO] [iat_publish-1]: process has finished cleanly [pid 11244]
1756483820.3662283 [INFO] [tts_subscribe-2]: process has finished cleanly [pid 11246]
